Gruntfile.js

```diff
@@ -845,8 +843,8 @@ module.exports = function(grunt) {
   grunt.registerTask('build', 'Build the static resources', [
     'exec:clean-build-dir',
     'copy:ddocs',
-    'build-node-modules',
     'build-common',
+    'build-node-modules',
```

Comment:
Switching the order is a hack. 
Running `browserify` post `dedupe` ends up not finding stuff.